bin/lib/fastly-extended.js

```diff
@@ -0,0 +1,133 @@
+var Fastly = require('fastly');
```

Comment:
Thanks. I think I'll make a separate PR for that, it's pretty specific to this task right now and could be generalized.